values.yaml

```diff
@@ -79,6 +79,20 @@ server:
     #   name: my-secret
     #   load: false # if true, will add to `-config-dir` to load by Consul
 
+  # Affinity Settings
+  # Overriding this will allow deployment to single node services such as
```

Comment:
What does an override look like? Can a user set the value to `null` or would they need to comment this option out? It would be good to add these details into the comment.

Reply:
Updated comment and provide details on the override.